src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -33,11 +33,13 @@ class XliffLintCommand extends Command
     protected static $defaultName = 'lint:xliff';
     protected static $defaultDescription = 'Lint an XLIFF file and outputs encountered errors';
 
-    private $format;
-    private $displayCorrectFiles;
+    private string $format;
+    private bool $displayCorrectFiles;
+    /** @var callable|null */
     private $directoryIteratorProvider;
+    /** @var callable|null */
     private $isReadableProvider;
```

Comment:
```suggestion
    private ?\Closure $directoryIteratorProvider;
    private ?\Closure $isReadableProvider;
```

Would it be possible to make the constructor convert the callables to closures like we did in other PRs?

Reply:
Good idea. Fixed in 3cf6702a3dccdcc19ab214869aba2dbeb22cd848